hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/snapshot/filter/ReclaimableFilter.java

```diff
@@ -167,11 +168,21 @@ private void initializePreviousSnapshotsFromChain(String volume, String bucket)
           previousOmSnapshots.add(null);
           previousSnapshotInfos.add(null);
         }
-
-        // NOTE: Getting volumeId and bucket from active OM.
-        // This would be wrong on volume & bucket renames support.
-        bucketInfo = ozoneManager.getBucketInfo(volume, bucket);
+      }
+      // NOTE: Getting volumeId and bucket from active OM.
+      // This would be wrong on volume & bucket renames support.
```

Comment:
this is not supported and is a design decision, so I think we can safely ignore it.